pandas/core/frame.py

```diff
@@ -3159,8 +3165,11 @@ def eval(self, expr, inplace=False, **kwargs):
         resolvers = kwargs.pop('resolvers', None)
         kwargs['level'] = kwargs.pop('level', 0) + 1
         if resolvers is None:
+            from pandas.core.computation.common import _get_column_resolvers
+
             index_resolvers = self._get_index_resolvers()
```

Comment:
this cannot be handled in _get_index_resolvers?

Reply:
index_resolvers returns indices. I need column resolvers as well. Moved this to a method in NDFrame, just like the index_resolvers.

Reply:
ok fair enough